helpdesk/templates/helpdesk/ticket_list.html

```diff
@@ -330,7 +331,13 @@ <h5 class="mb-0">
     {% else %}
     <script src="{% static 'helpdesk/vendor/timeline3/js/timeline.js' %}"></script>
     {% endif %}
+
     <script>
+        window.helpdesk_settings = {
+            LANGUAGE_CODE: "{{ helpdesk_settings.LANGUAGE_CODE|default:'en-US' }}",
+            NO_FOLLOWUP_TEXT: "{{ helpdesk_settings.TRANSLATIONS.No_followup_found|default:'No followup found' }}"
```

Comment:
Change this to:
`NO_FOLLOWUP_TEXT: "{% trans helpdesk_settings.TRANSLATIONS.No_followup_found|default:'No followup found' %}"`
It will ensure that the string can be translated.
Possibly change the settings based alternative for the english string to use something other than TRANSLATIONS since it is not actually translating the string - just providing an alternative string to the default.... maybe ALTERNATIVE_UI_STRINGS or something along those lines that provides a clearer idea of what it is doing.